src/resource/event/event.service.ts

```diff
@@ -42,7 +42,7 @@ export class EventService {
         return event;
     }
 
-    async fetchUserEvents(userId: number): Promise<ReturnEventDTO[]> {
+    async fetchUser(userId: number): Promise<ReturnEventDTO[]> {
```

Comment:
Wrong naming convention in my opinion, service method returns Events, not a User, I think something like "fetchByUser" would be more descriptive

Reply:
I agree